bay/images/Dockerfile.php

```diff
@@ -17,7 +17,8 @@
     rm -rf /blackfire
 
 COPY php/00-bay.ini /usr/local/etc/php/conf.d/
-RUN ep /usr/local/etc/php/conf.d/00-bay.ini
+
+COPY php/70-bay-php-config.sh /lagoon/entrypoints
```

Comment:
This needs to be an entrypoint otherwise downstream images won't be able to override with their env vars.